bundles/org.openhab.binding.mqtt.generic/README.md

```diff
@@ -216,6 +216,43 @@ mqttActions.publishMQTT("mytopic","myvalue", true)
 
 The retained argument is optional and if not supplied defaults to `false`.
 
+## Configuration via Text Files
+
+You find all examples in this blog post performed via Paper UI. You can of course still setup every aspect via text files, like in the following examples.
```

Comment:
I think the entire line can and should be removed.

Reply:
right, done